sdk/identity/identity/README.md

```diff
@@ -96,13 +96,7 @@ If interactive authentication can't be supported in the session, then the `-UseD
 
 #### Authenticate via Visual Studio Code
```

Comment:
I suggest removing this entire section.

Reply:
Should I also remove mentioning Visual Studio Code credential in [here](https://github.com/Azure/azure-sdk-for-js/blob/c286885ac0153d722410c8b868f14823e68a092f/sdk/identity/identity/README.md#authenticate-via-development-tools)?

Reply:
Yes, please remove that row too.